Documentation/building/windows-instructions.md

```diff
@@ -11,7 +11,9 @@ Visual Studio 2013 (Update 3 or later) or Visual Studio 2015 (Preview or later)
 
 The following free downloads are compatible:
 * [Visual Studio Community 2013 (with Update 3)](http://www.visualstudio.com/en-us/visual-studio-community-vs.aspx)
```

Comment:
I don't know much about Update 5 but if it isn't required as I don't believe it is I would just keep it as is it is for now.